.buildkite/longruns/pipeline.yml

```
@@ -301,6 +301,17 @@ steps:
           modules: climacommon/2025_01_27
         soft_fail: true
 
+      - label: "GPU AMIP with integrated land: ED only"
+        key: "amip_integrated_land_gpu"
+        command: "srun julia --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/run_amip.jl --config_file $CONFIG_PATH/amip_integrated_land.yml --job_id amip_integrated_land_gpu"
+        artifact_paths: "experiments/ClimaEarth/output/amip_integrated_land_gpu/artifacts/*"
+        env:
+          CLIMACOMMS_CONTEXT: "MPI"
+          CLIMACOMMS_DEVICE: "CUDA"
+        agents:
+          slurm_gpus: 1
+          slurm_mem: 16GB
```

```suggestion
      - label: "GPU AMIP with integrated land: ED only"
        key: "amip_integrated_land_gpu"
        command: "julia --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/run_amip.jl --config_file $CONFIG_PATH/amip_integrated_land.yml --job_id amip_integrated_land_gpu"
        artifact_paths: "experiments/ClimaEarth/output/amip_integrated_land_gpu/artifacts/*"
        env:
          CLIMACOMMS_DEVICE: "CUDA"
        agents:
          slurm_gpus: 1
          slurm_mem: 16GB
```
This is not running with MPI